torch/_inductor/codegen/triton.py

```diff
@@ -1429,7 +1429,9 @@ def load(self, name: str, index: sympy.Expr):
             if dtype == torch.bool:
                 # Workaround for https://github.com/openai/triton/issues/2151
                 # tl.load returns int8 when loading from pointer to int1
-                line += ".to(tl.int1)"
+                # NOTE: Currently causes hangs on bool UTs for ROCm
+                if torch.version.hip is None:
```

Comment:
NIT: can you merge this with the `if dtype == bool` condition above

Reply:
Sounds good to me. Updated the PR.